### ISSUE TITLE
[typescript-resolvers] Resolvers not pass ContextType to interface resolver

### ISSUE THREAD

**Issue by NateScarlet** [dotansimha/graphql-code-generator#4368]
**Describe the bug**

Can not use `Resolvers<MyContext>` to specify context for interface resolvers.

**To Reproduce**

1. My GraphQL schema:

```graphql
interface Node {
    id: ID!
}
```

2. My GraphQL operations:

```graphql
```

3. My `codegen.yml` config file:

```yml
generates:
  file.ts:
    config:
      useIndexSignature: true
      maybeValue: 'T extends PromiseLike<infer U> ? Promise<U | null | undefined> : T | null | undefined'
      contextType: unknown
    plugins:
      - 'typescript'
      - 'typescript-resolvers'
```

generates 

```typescript
export type Resolvers<ContextType = unknown> = ResolversObject<{
    Node?: NodeResolvers;
}>
```

**Expected behavior**

it should be

```typescript
export type Resolvers<ContextType = unknown> = ResolversObject<{
    Node?: NodeResolvers<ContextType>;
}>
```

**Environment:**

- OS: windows 7
- `@graphql-codegen/...`: 1.17.0
- NodeJS: 12.16.1

**Comment by dotansimha** [comment]
@NateScarlet You are right, thank you for reporting it.

**Comment by dotansimha** [comment]
@NateScarlet I pushed a fix in this PR: https://github.com/dotansimha/graphql-code-generator/issues/4368
You can use the alpha version `1.17.4-alpha-b439b445.0` for now, we'll release a stable version soon.

**Comment by dotansimha** [comment]
Fixed in `v1.17.4`
